conf/monitor/capture-statistics-template-ndjson.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: capturestatistics
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: capturestatistics-service-account
+#
+# Deployment of fluentd log collector
+#
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: capturestatistics
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: capturestatistics
+    template:
+      metadata:
+        labels:
+          app: capturestatistics
+      spec:
+        serviceAccountName: capturestatistics-service-account
+        containers:
+        - image: registry.fedoraproject.org/fedora
+          securityContext:
+            runAsUser: 0
+            privileged: true
+            allowPrivilegeEscalation: true
+          imagePullPolicy: Always
+          name: capturestatistics
+          command: ["/bin/bash"]
+          args:
+          - -c
+          - >
+            echo "=== setup-begin ===";
+            yum install -y procps;
+            yum install -y unzip;
+            curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl";
+            chmod +x kubectl;
+            unzip /go/bin/check-logs-sequence.zip;
+            echo "Collecting Statistics";
+            OUTPUT_FILE=/var/log/containers/stress.log;
+            echo -e "\nOutput log file is: $OUTPUT_FILE\n";
+            touch $OUTPUT_FILE;
+            ./check-logs-sequence -rf ndjson -f $OUTPUT_FILE -c ${number_of_log_lines_between_reports} -l ${check_log_level} &
+            echo "=== setup-end ===";
+            while true; do
```

Comment:
@jcantrill don't you think it makes sense to merge capture-statistics-template-ndjson.yaml and capture-statistics-template.yaml ??? the code is 99% equivalent can add `output_format` as another parameter and add if statement ... I think that this will be far better than two almost identical files.

Reply:
yes.  i agree but I was looking for the quick win to get something.  The fact the "report" for "vector top" is embedded in the template makes this difficult.  Maybe it makes more sense to capture it into a script and push it up to the image

Reply:
I will try to fix that in following PR